jsk_tools/CMakeLists.txt

```diff
@@ -41,12 +32,36 @@ if (CATKIN_ENABLE_TESTING)
   roslint_python(src/test_stdout.py)
   roslint_add_test()
   find_package(rostest REQUIRED)
+  macro(jsk_tools_add_rostest file)
+    list(APPEND jsk_tools_test_files ${file})
+    add_rostest(${file})
+  endmacro()
   if("$ENV{ROS_DISTRO}" STRGREATER "hydro")
     # FIXME: jsk_tools/test_topic_published.py does not work on hydro travis/jenkins
     # https://github.com/jsk-ros-pkg/jsk_common/pull/1293#issuecomment-164158260
-    add_rostest(test/test_topic_published.launch)
+    jsk_tools_add_rostest(test/test_topic_published.launch)
+    # FIXME: jsk_tools/sanity_lib.py does not work on hydro travis/jenkins
+    # https://github.com/jsk-ros-pkg/jsk_common/pull/1405#issuecomment-225218625
+    jsk_tools_add_rostest(test/python/sanity_lib.test)
   endif()
-  add_rostest(test/test_stdout.test)
+  jsk_tools_add_rostest(test/test_stdout.test)
```

Comment:
it is a macro. 
are you asking why it is necessary?
it is necessary to install only test files which works in the environment. 
some test files does not work on hydro, and it causes error in install testing in travis.